wix-data-v2/wix-data-v2/Items.service.json

```diff
@@ -1526,6 +1522,16 @@
           ],
           "doc": "Filter applied to the collection's data prior to running the aggregation. See [API Query Language](https://dev.wix.com/api/rest/getting-started/api-query-language#getting-started_api-query-language_the-filter-section) for information on how to structure a filter object."
         },
+        {
+          "name": "language",
```

Comment:
Seems ok to include, dev team needs to check how this module (data v1) is maintained.